### PR TITLE
np.testing import that is compatible with numpy 1.15

### PR DESCRIPTION
**Pull request by pllim**
This is to address astropy/astropy#7371

Caused by numpy/numpy#10850

### CODE REVIEW

File: scipy/stats/morestats.py [hunk 1]
```diff
@@ -3,13 +3,18 @@
 import math
 import warnings
 from collections import namedtuple
+from distutils.version import LooseVersion
```
Comment (pv) [inline scipy/stats/morestats.py]: from scipy._lib._version import NumpyVersion

File: scipy/stats/morestats.py [hunk 2]
```diff
@@ -3,13 +3,18 @@
...
 
 import numpy as np
 from numpy import (isscalar, r_, log, around, unique, asarray,
                    zeros, arange, sort, amin, amax, any, atleast_1d,
                    sqrt, ceil, floor, array, compress,
                    pi, exp, ravel, count_nonzero, sin, cos, arctan2, hypot)
-from numpy.testing.decorators import setastest
+# https://github.com/numpy/numpy/pull/10850
```
Comment (pv) [inline scipy/stats/morestats.py]: Link not needed I think.

**Comment by pllim** [comment]
Thanks!

**Comment by pv** [comment]
On second thought, we can simply remove it, since this is some nose stuff.

**Comment by pllim** [comment]
> we can simply remove it

Certainly a much easier fix that way! :smile:

**Comment by pv** [comment]
thanks, merged

**Pull request merged**